src/read.c

```diff
@@ -3081,6 +3081,11 @@ avifResult avifDecoderParse(avifDecoder * decoder)
             // probably exif or some other data
             continue;
         }
+        if (item->auxForID) {
+            // NON-STANDARD: Allow subimages to not have an ispe property
+            // See: https://crbug.com/1245673
+            continue;
+        }
```

Comment:
At the end of avifDecoderItemValidateAV1(), we have this code:

```
    if (strictFlags & AVIF_STRICT_CLAP_VALID) {
        const avifProperty * clapProp = avifPropertyArrayFind(&item->properties, "clap");
        if (clapProp) {
            const avifProperty * ispeProp = avifPropertyArrayFind(&item->properties, "ispe");
            if (!ispeProp) {
                avifDiagnosticsPrintf(diag,
                                      "[Strict] Item ID %u is missing an ispe property, so its clap property cannot be validated",
                                      item->id);
                return AVIF_RESULT_BMFF_PARSE_FAILED;
            }

            ...
        }
    }
```

Should we also allow the auxiliary images to not have an ispe property there?

Reply:
I don't believe auxiliary images will have a `clap` property (it will typically only be on the primary item), so this should be benign. I'd rather keep this strict.